src/collections/configure/generative.ts

```diff
@@ -262,7 +269,14 @@ export default {
   ): ModuleConfig<'generative-google', GenerativeGoogleConfig | undefined> => {
     return {
       name: 'generative-google',
-      config,
+      // Do not populate config key if config === undefined.
+      config: config
+        ? {
+            ...config,
+            // Only create modelId key if either config.model or config.modelId are defined.
+            ...(config?.modelId || config?.model ? { modelId: config?.model ?? config?.model } : undefined),
```

Comment:
This syntax is fairly dense. My intention was to copy the model name into the `modelId` variable (for b/c) but preserve the previous behavior for `config === undefined`